src/xcode/ENA/ENA/Resources/Localization/tr.lproj/Localizable.stringsdict

```diff
@@ -1,6 +1,23 @@
-<?xml version="1.0" encoding="UTF-8"?><!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>ExposureDetection_Explanation_Text_High_DaysSinceLastExposure</key>
+	<dict>
+		<key>NSStringLocalizedFormatKey</key>
+		<string>%#@VARIABLE@</string>
+		<key>VARIABLE</key>
+		<dict>
+			<key>NSStringFormatSpecTypeKey</key>
+			<string>NSStringPluralRuleType</string>
+			<key>NSStringFormatValueTypeKey</key>
+			<string>u</string>
+			<key>one</key>
+			<string>En son %u gün önce, COVID-19 tanısı konan en az bir kişiyle daha uzun süreyle ve yakın mesafeden maruz kaldığınız için enfeksiyon riskiniz daha yüksektir.</string>
+			<key>other</key>
+			<string>En son %u gün önce, COVID-19 tanısı konan en az bir kişiyle daha uzun süreyle ve yakın mesafeden maruz kaldığınız için enfeksiyon riskiniz daha yüksektir.</string>
```

Comment:
Hi, 
it should be like this for one:
`En son bir gün önce, COVID-19 tanısı konan en az bir kişiye uzun süreyle yakın mesafeden maruz kaldığınız için enfeksiyon riskiniz yüksektir.`
And for other:
`En son %u gün önce, COVID-19 tanısı konan en az bir kişiye uzun süreyle yakın mesafeden maruz kaldığınız için enfeksiyon riskiniz yüksektir.`

Reply:
thank you, I implemented your proposals!